src/System.Transactions.Local/src/System/Transactions/TransactionManager.cs

```diff
@@ -269,18 +267,7 @@ public static void RecoveryComplete(Guid resourceManagerIdentifier)
         private static object s_classSyncObject;
 
         // Helper object for static synchronization
-        private static object ClassSyncObject
-        {
-            get
-            {
-                if (s_classSyncObject == null)
-                {
-                    object o = new object();
-                    Interlocked.CompareExchange(ref s_classSyncObject, o, null);
-                }
-                return s_classSyncObject;
-            }
-        }
+        private static object ClassSyncObject => LazyInitializer.EnsureInitialized(ref s_classSyncObject);
```

Comment:
Yes. It is being used at line 39, 51, 81 and 121 in the get / set which are not type of lazy init...